### DISCUSSION TITLE
How to properly set up thumbnails for social media posts?

### DISCUSSION THREAD

**Discussion by DavSanchez** [srid/emanote#562] [Q&A]
Hi!

Just shared a recent article on [my emanote-based blog](https://blog.tech.davidslt.es/blog/2024-11-13-mapeando-estructuras.html) ([code](https://github.com/DavSanchez/davsanchez.github.io/blob/e06bc19fb65a57f0a8e2b216c429658e631760f7/blog/2024-11-13-mapeando-estructuras.md?plain=1#L5-L6)) but the first image does not seem to load on social media previews for the article. Tried to set up `page.image` in the article heading as suggested in PR https://github.com/srid/emanote/pull/499 but does not seem to work (unless I'm not understanding well where to place it).

Is there an standardised way to add this preview image?

Thanks!

**Comment by srid** [comment]
The URLs don't match?

Image is hosted at `/blog/img/functor-diagram.png`

But your YAML points to `img/functor-diagram.png`

![image](https://github.com/user-attachments/assets/3c8a9eae-e624-4889-af71-cb7e92da33cb)

**Reply by DavSanchez** [reply]
Hmm thought they could be relative to the written content. Thanks! I’ll try it and get back with an update.

**Reply by srid** [reply]
I believe the base URL is used to resolve it,

![image](https://github.com/user-attachments/assets/53447771-e9e1-4a6b-b445-1ab81dc0c268)
